src/types/multi-select.ts

```diff
@@ -14,3 +15,164 @@ export interface MultiselectFilterFunctionParams {
 }
 
 export type DropdownFooterTextPosition = 'sticky' | 'static'
+
+export interface MultiselectProps {
+  /**
+   * The current value of the multiselect (v-model).
+   * @default []
+   */
+  modelValue?: string[]
+
+  /**
+   * The label for the multiselect.
+   * @default ''
+   */
+  label?: string
+
+  /**
+   * The help text displayed below the multiselect.
+   * @default ''
+   */
+  help?: string
+
+  /**
+   * Indicates whether there is an error state.
+   * @default false
+   */
+  error?: boolean
+
+  /**
+   * Additional attributes for the label element.
+   * @default {}
+   */
+  labelAttributes?: Record<string, any>
```

Comment:
Rebase to `main` and we can add a `LabelAttributes` interface based on [`LabelProps`](https://github.com/Kong/kongponents/blob/main/src/types/label.ts) like [`TooltipAttributes`](https://github.com/Kong/kongponents/blob/main/src/types/tooltip.ts#L62).